exercises/javascript-1-a/javascript-1-a.js

```diff
@@ -0,0 +1,33 @@
+// @ts-check
+
+/**
+ * Re-arranges the input array by moving an item from from, to to.
```

Comment:
I found the "from from" and "to to" bit a bit weird to read. Maybe something like this?

```suggestion
 * Re-arranges the input array by moving an item from one position to another position.
```